Gopkg.lock

```diff
@@ -837,22 +845,23 @@
     "storage/v1beta1",
   ]
   pruneopts = "UT"
-  revision = "05914d821849570fba9eacfb29466f2d8d3cd229"
-  version = "kubernetes-1.13.1"
+  revision = "74b699b93c15473932b89e3d1818ba8282f3b5ab"
+  version = "kubernetes-1.13.3"
```

Comment:
Kinda ambivalent towards locking into a Kubernetes dependency without pinning it. Will defer to @justaugustus on that.

If this is agreed upon, Can we change to `kubernetes-1.13.4`